source/statements.tex

```diff
@@ -820,7 +820,8 @@
 \indextext{\idxcode{return}!constructor and}%
 \indextext{\idxcode{return}!constructor and}%
 A \tcode{return} statement with an operand of type \tcode{void} shall be used only
-in a function whose return type is \cv{}~\tcode{void}.
+in a function whose return type is \cv{}~\tcode{void}, in which case the operand is
+a discarded-value expression\iref{expr.context}.
```

Comment:
I don't think we need/want to say this here. If you have an operand of void type, nothing needs to be discarded to start with.